website/pages/blog/complete-guide-to-graphql-scalar.mdx

```diff
@@ -0,0 +1,423 @@
+---
+title: Complete Guide to GraphQL Scalar
```

Comment:
I'm still trying to decide whether to include the following:

- Link to https://scalars.graphql.org/ explaining what it's for
- (Lack of) Client type for `graphql-scalars`